spec/support/form_helpers.rb

```diff
@@ -18,6 +18,7 @@ def fill_in_activity_form(
     total_applications: "12",
     total_awards: "5",
     programme_status: "07",
+    country_delivery_partners: "National Council for the State Funding Agencies (CONFAP)",
```

Comment:
Could we provide this as an array, to make it match the underlying data type, or does it make filling in the form harder further down?

Reply:
I think if we provide this as an array, it will store it as an array inside an array